### ISSUE TITLE
Is it possible to automatically close the terminal panel that runs fzf after completing or canceling a search session?

### ISSUE THREAD

**Issue by taquangtrung** [rlivings39/vscode-fzf-quick-open#50]
Hi,

Thanks for making such a useful extension!

I want to ask how I can make it automatically close the terminal panel that runs `fzf` after completing or canceling a search session?

Currently, when I cancel a search (by `Esc`) or visit a searched pattern (by `Enter`), the terminal panel that runs `fzf` still open. 

FYI, I notice that in another extension https://github.com/tomrijndorp/vscode-finditfaster, it can automatically close the terminal when I press `Esc` or `Enter`.

Thank you!

**Comment by tru** [comment]
Yeah hiding or exiting the terminal after selecting a file would be awesome.
